assets/js/atomic/blocks/product-elements/rating/block.tsx

```diff
@@ -40,12 +40,35 @@ const getRatingCount = ( product: ProductResponseItem ) => {
 	return Number.isFinite( count ) && count > 0 ? count : 0;
 };
 
+const getStarStyle = ( rating: number ) => ( {
+	width: ( rating / 5 ) * 100 + '%',
+} );
+
+const NoRating = ( { parentClassName }: { parentClassName: string } ) => {
+	const starStyle = getStarStyle( 0 );
+
+	return (
+		<div
+			className={ classnames(
+				'wc-block-components-product-rating__norating-container',
+				`${ parentClassName }__product-rating__norating-container`
+			) }
+		>
+			<div
```

Comment:
Do we need to have divs within divs to achieve this? I feel like we can just have one div and have the two spans next to each other?

Reply:
I prefered this approach to put easily on the same line height.

Reply:
`wc-block-components-product-rating__norating` div, we applied the custom font to render the start. Adding `flex` to the parent container could influence the other cases. Keep in mind that this is just a temporary solution.